app/generators/cinema_halls/repository.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module CinemaHalls
```

Comment:
Again - how this class is connected to `generators` directory where you created it?